src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@import './variables';
+
+.product {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  box-sizing: border-box;
+  justify-content: space-between;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background: $white-color;
+  transition: transform $product-transition linear;
+  cursor: pointer;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    padding-bottom: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    padding: 0;
```

Comment:
You can reset all default styles in `index.scss` or make a separate `reset.scss` file
```suggestion
```